.gitignore

```diff
@@ -9,14 +9,17 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-
 # Added by cargo
-
 /target
 
+# Local LLVM framework.
 llvm/
 
-# Added so we can have our own specific development scripts
-scripts/
```

Comment:
Instead of removing this, could we change it to maybe `dev-scripts/` that is still gitignored? The point of having a place to hold scripts for development that are only useful to the person who wrote them still holds imo even if we now have platform independent ones too

Reply:
I just tested something and I found another way to keep a local gitignored scripts folder: just having a folder with a `.gitignore` that ignores itself seems to work, and another advantage is that it doesn't have to be named `dev-scripts/`.

What do you think?